packages/react-query/src/shared/hooks/createHooksInternal.tsx

```diff
@@ -484,6 +485,8 @@ export function createRootHooks<
   ): UseTRPCSuspenseInfiniteQueryResult<unknown, TError, unknown> {
     const hookResult = useInfiniteQuery(path, input, {
       ...opts,
+      // FIXME: How should we handle suspense hooks?
+      // @ts-expect-error - whatever?
       suspense: true,
       enabled: true,
       throwOnError: true,
```

Comment:
FYI, we also don't default to `throwOnError: true`. details on that here: https://tanstack.com/query/v5/docs/react/guides/suspense#throwonerror-default

Reply:
Let's use the same behavior as RQ here then!